start.cmd

```diff
@@ -3,7 +3,8 @@
 echo.
 echo Restoring backend python packages
 echo.
-call python -m pip install -r requirements.txt
+
+call poetry install
```

Comment:
I should probably update a few other scripts as well, just realized this might not actually be complete.  I'll take a deeper look at the scripts after discussing the overall approach.